ydb/library/yql/minikql/comp_nodes/mkql_grace_join.cpp

```diff
@@ -767,6 +767,12 @@ class TGraceJoinSpillingSupportState : public TComputationValue<TGraceJoinSpilli
 
             bool isYield = FetchAndPackData(ctx);
             if (ctx.SpillerFactory && IsSwitchToSpillingModeCondition()) {
+                const auto used = TlsAllocState->GetUsed();
+                const auto limit = TlsAllocState->GetLimit();
+
+                YQL_LOG(INFO) << "yellow zone reached " << (used*100/limit) << "%=" << used << "/" << limit;
+                YQL_LOG(INFO) << "switching Memory mode to Spilling";
```

Comment:
Is it possible to add task id here?

Reply:
YQL_LOG adds tid/etc. Complete log line looks like
```
2024-05-29 16:04:59.956 INFO  dqrun(pid=103200, tid=0x000075689560B640) [default] mkql_grace_join.cpp:773: yellow zone reached 79%=41530384/52428800
```

Reply:
Yes, but tid/pid doesn't represent the id of the task. 
Tbh, I'm not sure if it's possible to add `TaskId` here... Maybe it is stored somewhere in `ctx`?

Without the taskId, we only know the fact that spilling was enabled somewhere

Reply:
I have not found anything in `ctx` with `TaskId` in name (but maybe I'm looking at wrong parts?)

Reply:
decided to add task id in separate task